updating/update-using-custom-machine-config-pools.adoc

```diff
@@ -0,0 +1,220 @@
+[id="update-using-custom-machine-config-pools"]
+= Use of customized machine config pools during update
```

Comment:
Perhaps we can keep it `Use of custom machine config pools during update`

Reply:
customized is a better word than custom IMO. However these will be sorted when doc peer review happens. So I am not worried about these.

Reply:
@LalatenduMohanty @sinnykumari We use the term _custom machine config pools_ elsewhere in the docs.

Reply:
@mburke5678 sounds good to me.